symphony/bdk/core/service/message_service.py

```diff
@@ -0,0 +1,366 @@
+                'auth': [],
+                'endpoint_path': '/v4/message/blast',
+                'operation_id': 'v4_message_blast_post',
+                'http_method': 'POST',
+                'servers': None,
+            },
+            params_map={
+                'all': [
+                    'session_token',
+                    'sids',
+                    'key_manager_token',
+                    'message',
+                    'data',
+                    'version',
+                    'attachment',
+                    'preview',
+                ],
+                'required': [
+                    'session_token',
+                    'sids',
+                ],
+                'nullable': [
+                ],
+                'enum': [
+                ],
+                'validation': [
+                ]
+            },
+            root_map={
+                'validations': {
+                },
+                'allowed_values': {
+                },
+                'openapi_types': {
+                    'session_token':
+                        (str,),
+                    'sids':
+                        ([str],),
+                    'key_manager_token':
+                        (str,),
+                    'message':
+                        (str,),
+                    'data':
+                        (str,),
+                    'version':
+                        (str,),
+                    'attachment':
+                        ([file_type],),
+                    'preview':
+                        ([file_type],),
+                },
+                'attribute_map': {
+                    'session_token': 'sessionToken',
+                    'sids': 'sids',
+                    'key_manager_token': 'keyManagerToken',
+                    'message': 'message',
+                    'data': 'data',
+                    'version': 'version',
+                    'attachment': 'attachment',
+                    'preview': 'preview',
+                },
+                'location_map': {
+                    'session_token': 'header',
+                    'sids': 'form',
+                    'key_manager_token': 'header',
+                    'message': 'form',
+                    'data': 'form',
+                    'version': 'form',
+                    'attachment': 'form',
+                    'preview': 'form',
+                },
+                'collection_format_map': {
+                    'sids': 'csv',
+                }
+            },
+            headers_map={
+                'accept': [
+                    'application/json'
+                ],
+                'content_type': [
+                    'multipart/form-data'
+                ]
+            },
+            api_client=api_client,
+            callable=__v4_message_create_post
+        )
+
+
+class MessageService:
+    def __init__(self, messages_api: MultiAttachmentsMessagesApi,
```

Comment:
Just to check. I am having some confusion. 
What problem does the `MultiAttachmentsMessagesApi` solve that the generated `MessagesApi` has? I can't seem to find it on the java BDK.
Is it related to some new specs?

Reply:
The generated `MessagesApi` doesn't allow to send a message with multiple attachments which is allowed by the api provided by the agent. That's why I create the `MultiAttachmentsMessagesApi` which accepts the attachment field as a list of file.